rt_utils/utils.py

```diff
@@ -2,12 +2,38 @@
 from pydicom.uid import PYDICOM_ROOT_UID
 from dataclasses import dataclass
 
-class SOPClassUID():
-    RTSTRUCT_IMPLEMENTATION_CLASS = PYDICOM_ROOT_UID # TODO find out if this is ok
+COLOR_PALLET = [
```

Comment:
```suggestion
COLOR_PALETTE= [
```

Reply:
what kind of colors are these btw

Reply:
OH they are the ones from MIM. Carlos gave them to me #11

Reply:
sweet